packages/backend-common/src/reading/tree/ZipArchiveResponse.test.ts

```diff
@@ -35,24 +35,25 @@ const archiveWithMaliciousEntry = fs.readFileSync(
 );
 
 describe('ZipArchiveResponse', () => {
-  beforeEach(() => {
-    mockFs({
-      '/test-archive.zip': archiveData,
-      '/test-archive-with-extra-root-dir.zip': archiveDataWithExtraDir,
-      '/test-archive-corrupted.zip': archiveDataCorrupted,
-      '/test-archive-malicious.zip': archiveWithMaliciousEntry,
-      '/tmp': mockFs.directory(),
+  const sourceDir = MockDirectory.create();
+  const targetDir = MockDirectory.create();
+
+  beforeAll(() => {
+    sourceDir.setContent({
```

Comment:
feeling `contents` is a pretty good field to have on `create`, maybe it could even do `beforeEach(() => this.setContent(c))` internally which would make it consistent across tests automatically

Reply:
Think there's a risk of too much magic at that point tbh. Feeling it's not obviously clear that it would have that behavior when you're reading the test code. The existing `afterAll` calls are just for cleanup and doesn't really effect the tests.